Tools/clinic/clinic.py

```diff
@@ -221,6 +221,23 @@ def c_repr(s: str) -> str:
     return '"' + s + '"'
 
 
+def wrapped_c_string_literal(
+        text: str,
+        *,
+        width: int = 72,
+        suffix: str = '',
+        initial_indent: int = 0,
+        subsequent_indent: int = 4
```

Comment:
It seems all call sites specify a value for the `subsequent_indent` parameter. Maybe it should be required, instead of optional?

```suggestion
        subsequent_indent: int,
        width: int = 72,
        suffix: str = '',
        initial_indent: int = 0,
```

Reply:
Hm, perhaps.